src/main/java/org/entando/kubernetes/security/AuthorizationFilter.java

```diff
@@ -0,0 +1,126 @@
+package org.entando.kubernetes.security;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.regex.Pattern;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.entando.kubernetes.model.web.response.SimpleRestResponse;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class AuthorizationFilter extends HttpFilter {
+
+    public static final Pattern SERVLET_PATH_REGEX = Pattern.compile("^((/components)|(/registries)).*");
```

Comment:
what about `/bundles`?

Reply:
`/bundles` is exposing only services in `GET` and we are allowing read accesses